spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/AbstractPubSubSender.java

```diff
@@ -18,7 +18,7 @@
 
```

Comment:
I can't point you to the line of code, but this class is still `AbstractPubSubSender`

I think you can just make a search through the project for the `*.java` files and the `PubSub` criteria match case:

![image](https://user-images.githubusercontent.com/1071590/27915876-7f15a1ee-6235-11e7-9b48-12eb206fb335.png)